src/codeu/chat/client/commandline/Chat.java

```diff
@@ -111,6 +116,28 @@ public void invoke(Scanner args) {
         System.out.println("    Exit the program.");
       }
     });
+    
+    
+    // INFO 
+    //
+    // Gives user information on the server 
+    // Added only for TimeUp which gives server's start time
+    // added by Julia 5/19 
+    
+    panel.register("info", new Panel.Command() {
+    	  @Override
+    	  public void invoke(Scanner args) {
+    	    final ServerInfo info = context.getInfo();
+    	    if (info == null) {
+    	    	System.out.println("ERROR: Failed to retrieve server information"); 
+    	    } else {
+    	    	System.out.format(
```

Comment:
I think this is why you're getting a time instead of a duration. I think you'd want to do some kind of subtraction here?

The other question is: why are you getting such a weird date for the start time? Just a guess: maybe something to do with `int` and `long` conversion? Try printing out what the server is generating in addition to what the client is receiving here?